tests/system/conftest.py

```diff
@@ -155,6 +155,8 @@ def data_instance_populated(
         )
         operation = instance.create(clusters=[cluster])
         operation.result(timeout=30)
+    else:
+        instance = admin_client.instance(data_instance_id)
```

Comment:
This seems similar to ln 152, could that just be pulled above the if?

Reply:
47ec1e9